### PR TITLE
 Fix #1768 Creating GIF to show process of creating badges similar to master branch 

### LINKED ISSUE #1768: Creating GIF to show process of creating badges similar to master branch
**I'm submitting a ...**
- [x] feature request

**Current behaviour:**

There is no gif showing updated process of generating badgeyay in development branch .

Will work on issue??
Yes

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1768

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Adding a demo gif.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ The user can:
 - Choose the background of the badges and upload the logo and the background image
 - Upload a CSV file or manually enter CSV data as: name, type of attendee, designation, nick/handle, organization/project
 
+##Demo
```
Comment (gabru-md) [inline README.md]: space is required here so that the markdown format can render it.
thanks you! please make the necessary changes and then we a

Comment (sk9331657) [reply README.md]: Done.

**Comment by sk9331657** [comment]
@gabru-md Please review.

**Comment by sk9331657** [comment]
@ParthS007  @gabru-md Done change required.

**Comment by sk9331657** [comment]
@gabru-md  @ParthS007  @yashLadha  Please review and merge this.

**Pull request merged**